src/components/Palette/Palette.jsx

```diff
@@ -15,24 +17,42 @@ const palette = [
   "#E8EAED",
 ];
 
-export const Palette = () => {
+export const Palette = ({ dataItem, type, setNoteInfo }) => {
   const [isHidden, setIsHidden] = useState(true);
+  const { notes, noteData, noteDispatch } = useNotes();
+  // console.log(noteData);
```

Comment:
There are `console.log()` s commented in almost all the files. Remove them after debugging